.travis.yml

```diff
@@ -19,3 +21,16 @@ matrix:
       env: DJANGO_VERSION=2.0
     - python: "2.7"
       env: DJANGO_VERSION=2.0
+    - python: "3.6"
+      env: DJANGO_VERSION=1.4
+    - python: "3.6"
+      env: DJANGO_VERSION=1.5
+    - python: "3.6"
+      env: DJANGO_VERSION=1.10
+    - python: "3.7"
+      env: DJANGO_VERSION=1.4
+    - python: "3.7"
+      env: DJANGO_VERSION=1.5
+    - python: "3.7"
+      env: DJANGO_VERSION=1.10
+      
```

Comment:
Strangely, Github still says an empty line is missing. Does line 36 have empty spaces and is not a clear line?
(I'm aware this is picky)